docs-2.0/3.ngql-guide/8.clauses-and-options/ttl-options.md

```diff
@@ -0,0 +1,103 @@
+# TTL
+
+TTL indicates time to live. Use the [TTL options](#ttl_options) to specify a timeout for a vertex property or an edge property. Once the timeout is reached, the property expires.
```

Comment:
for a property? Because TTL is not for a single vertex, it's for a tag or edge type.